src/nnet3/online-nnet3-decodable-simple.h

```diff
@@ -102,6 +102,7 @@ class DecodableNnet3SimpleOnline: public DecodableInterface {
   /// Indices are one-based!  This is for compatibility with OpenFst.
   virtual int32 NumIndices() const { return trans_model_.NumTransitionIds(); }
 
+  virtual int32 FrameSubsamplingFactor() const { return opts_.frame_subsampling_factor; }
```

Comment:
you can remove the 'virtual' here, since it isn't part of the interface.

Overall this is a little bit of a hack, but if I think of a more general way to do this we can do it later on.